helpers/deploymentConfig.ts

```diff
@@ -2722,6 +2754,12 @@ export const globalConfig: NetworkConfig = {
             supplySpeeds: ["712962962962"], // 0.154 WSTETH over 30 days (216000 blocks)
             borrowSpeeds: ["0"],
           },
+          {
+            asset: "USDC",
+            markets: ["weETH"],
+            supplySpeeds: ["5787037037000000"], // 5,000 USDC for 30 days (864000 blocks)
```

Comment:
```suggestion
            supplySpeeds: ["23148148148148148"], // 5,000 USDC for 30 days (216000 blocks)
```

On Ethereum, we assume there are 1 block every 12 seconds